src/internal/opts.rs

```diff
@@ -72,4 +72,7 @@ pub struct FdOptions {
 
     /// Constraints on last modification time of files
     pub time_constraints: Vec<TimeFilter>,
+
+    /// Whether to print errors on stdout
```

Comment:
This does not disable all errors. So the docstring should be something like: "Whether or not to display filesystem errors"